index.html

```diff
@@ -117,20 +117,30 @@ <h2>API</h2>
 import * as parsel from "./parsel.js";
 self.parsel = parsel; // so that people can experiment in the console
 
+const url = new URL(location);
+if(url.searchParams.get("selector")) {
+    let selector = url.searchParams.get("selector");
+    selectorText.value = selector || selectorText.value; // set default value if the selector is empty
+    url.searchParams.set("selector", selectorText.value);
+}
+
 (tryout.oninput = e => {
-	tokens.textContent = JSON.stringify(parsel.tokenize(selectorText.value), null, "\t");
-	Prism.highlightElement(tokens);
+        url.searchParams.set("selector", selectorText.value);
```

Comment:
Better reassign `url` here (before this line), otherwise if there's any hash it will be reset to whatever it was when the page loaded.